flake8_quotes/__init__.py

```diff
@@ -30,13 +30,18 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', action='store', dest='inline_quotes',
+                          help='Deprecated alias for `--inline-quotes`')
+        parser.add_option('--inline-quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append('quotes')
+        parser.config_options.extend(['quotes', 'inline-quotes'])
 
     @classmethod
     def parse_options(cls, options):
-        cls.quotes = cls.QUOTES[options.quotes]
+        if not hasattr(options, 'inline_quotes') and hasattr(options, 'quotes'):
```

Comment:
Further events: `hasattr(options, 'quotes')` seems to always be `True` so we should prob use `if options.quotes` to check for falsy